### PR TITLE
Wimpie/information security

### PR DESCRIPTION
**Pull request by Wimmas**
Added auth policies to endpoints where they were missing. Please check the ones that I used.

### CODE REVIEW

**Review by dvanhout89 — DISMISSED** [review]

File: RR.App/Controllers/HRIS/ChartsController.cs [hunk 1]
```diff
@@ -17,6 +18,8 @@ public ChartsController(IChartService chartService)
         _chartService = chartService;
     }
 
+
```
Comment (bseptember) [inline RR.App/Controllers/HRIS/ChartsController.cs:21]: Refactor: empty line or white space.

**Review by bseptember — DISMISSED** [review]

File: RR.App/Controllers/HRIS/ChartsController.cs [hunk 2]
```diff
@@ -40,6 +43,8 @@ public async Task<IActionResult> GetAllCharts()
         }
     }
 
+
```
Comment (tbddorothy) [inline RR.App/Controllers/HRIS/ChartsController.cs:46]:
```suggestion
```

File: RR.App/Controllers/HRIS/ChartsController.cs [hunk 3]
```diff
@@ -56,6 +61,8 @@ public async Task<IActionResult> GetAllCharts()
...
```
Comment (tbddorothy) [inline RR.App/Controllers/HRIS/ChartsController.cs:64]:
```suggestion
```

File: RR.App/Controllers/HRIS/ChartsController.cs [hunk 4]
```diff
@@ -70,6 +77,8 @@ public async Task<IActionResult> GetChartData([FromQuery] List<string> dataTypes
...
```
Comment (tbddorothy) [inline RR.App/Controllers/HRIS/ChartsController.cs:80]:
```suggestion
```

File: RR.App/Controllers/HRIS/ChartsController.cs [hunk 5]
```diff
@@ -86,6 +95,8 @@ public async Task<IActionResult> UpdateChartData(ChartDto chartDto)
...
```
Comment (tbddorothy) [inline RR.App/Controllers/HRIS/ChartsController.cs:98]:
```suggestion
```

File: RR.App/Controllers/HRIS/ChartsController.cs [hunk 6]
```diff
@@ -100,6 +111,8 @@ public async Task<IActionResult> DeleteChart([FromQuery] int chartId)
...
```
Comment (tbddorothy) [inline RR.App/Controllers/HRIS/ChartsController.cs:114]:
```suggestion
```

File: RR.App/Controllers/HRIS/ChartsController.cs [hunk 7]
```diff
@@ -120,6 +133,8 @@ public IActionResult GetColumns()
...
```
Comment (tbddorothy) [inline RR.App/Controllers/HRIS/ChartsController.cs:136]: Please fix the unnecessary empty spaces, I'm not gonna comment on all of them

**Review by bseptember — APPROVED** [review]

**Review by Juanro-Minne — APPROVED** [review]

**Review by tbddorothy — APPROVED** [review]

**Pull request merged**